ast/functions.go

```diff
@@ -330,7 +330,100 @@ type FuncCallExpr struct {
 
 // Restore implements Node interface.
 func (n *FuncCallExpr) Restore(ctx *RestoreCtx) error {
-	return errors.New("Not implemented")
+	ctx.WriteKeyWord(n.FnName.O)
+	ctx.WritePlain("(")
+	switch n.FnName.L {
+	case "convert":
+		if err := n.Args[0].Restore(ctx); err != nil {
+			return errors.Annotatef(err, "An error occurred while restore FuncCastExpr.Expr")
+		}
+		ctx.WriteKeyWord(" USING ")
+		ctx.WritePlain(n.Args[1].GetType().Charset)
```

Comment:
@leoppro Should this use `WriteKeyWord`? The character set is case-insensitive.

```sql
select convert('1234' using UtF8Mb4);
```

Reply:
ok, @buzzers 
please revise this line then we will merge the pr.

Reply:
PTAL @kennytm